src/history/convert_pdfs.py

```diff
@@ -44,19 +46,19 @@ def convert(pdf_files: list[str]) -> None:
         # Check if the file extension is .pdf
         if file.endswith(".pdf"):
             # Delete the file
-            os.remove(os.path.join(TMP_DIRECTORY, file))
+            Path(Path(TMP_DIRECTORY) / file).unlink()
```

Comment:
Since `Path(TMP_DIRECTORY) / file` is already a `Path`, is there a need to wrap it in a `Path()` constructor again?